packages/web-config-server/src/apiV1/dataBuilders/generic/count/countEventsPerPeriodByDataValue2.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Tupaia Config Server
+ * Copyright (c) 2019 Beyond Essential Systems Pty Ltd
+ */
+
+/* eslint-disable max-classes-per-file */
+
+import moment from 'moment';
+import { DataBuilder } from '/apiV1/dataBuilders/DataBuilder';
+import { DataPerPeriodBuilder } from 'apiV1/dataBuilders/DataPerPeriodBuilder';
+import { groupEventsByPeriod } from '@tupaia/dhis-api';
+import { getAllDataElementCodes } from '../../helpers';
+
+/**
+ * @abstract
+ */
+class DataByValueBuilder2 extends DataBuilder {
```

Comment:
This is bad but intentional. Cleanup split off to another card to make this one easier to review. Cleanup card: #2203